### ISSUE TITLE
MindCV Roadmap - feature requestment

### ISSUE THREAD

**Issue by SamitHuang** [mindspore-ecosystem/mindcv#238]
We are open to collecting valuable new models and algorithms from users! 

You are welcome to leave a comment to suggest a new feature/model/algorithm for MIndCV. 
You can also vote or dis-vote a suggestion.

**Comment by Ash-Lee233** [comment]
beit-base：
https://github.com/microsoft/unilm/tree/master/beit
https://arxiv.org/pdf/2106.08254v2.pdf

**Comment by Ash-Lee233** [comment]
- [ ] beit-base: https://github.com/microsoft/unilm/tree/master/beit

**Comment by Ash-Lee233** [comment]
# Models
- [ ] [beitv2](https://github.com/microsoft/unilm/tree/master/beit2)
- [ ] [CoAtNet](https://github.com/rwightman/pytorch-image-models/blob/main/timm/models/maxxvit.py)
- [ ] [convnextv2](https://github.com/facebookresearch/convnext-v2)
- [ ] [cspnet](https://github.com/rwightman/pytorch-image-models/blob/main/timm/models/cspnet.py)
- [ ] [deit](https://github.com/rwightman/pytorch-image-models/blob/main/timm/models/deit.py)
- [ ] [dla](https://github.com/rwightman/pytorch-image-models/blob/main/timm/models/dla.py)
- [ ] [hardcorenas](https://github.com/rwightman/pytorch-image-models/blob/main/timm/models/hardcorenas.py)
- [ ] [inception_resnet_v2](https://github.com/rwightman/pytorch-image-models/blob/main/timm/models/inception_resnet_v2.py)
- [ ] [maxvit](https://github.com/google-research/maxvit)
- [ ] [mobilevit-v2](https://github.com/apple/ml-cvnets)
- [ ] [simclr](https://github.com/google-research/simclr)
- [ ] [tnt](https://github.com/rwightman/pytorch-image-models/blob/main/timm/models/tnt.py)
- [ ] [tresnet](https://github.com/rwightman/pytorch-image-models/blob/main/timm/models/tresnet.py)
- [ ] [twins](https://github.com/rwightman/pytorch-image-models/blob/main/timm/models/twins.py)
- [ ] [CLIP](https://github.com/openai/CLIP)
- [ ] [ViTDet](https://github.com/facebookresearch/detectron2/tree/main/projects/ViTDet)
- [ ] [vovnet](https://github.com/rwightman/pytorch-image-models/blob/main/timm/models/vovnet.py)
- [ ] [wideresnet](https://github.com/szagoruyko/wide-residual-networks)

**Comment by Ash-Lee233** [comment]
# Augmentation

- [ ] AugMix
- [ ] 3-Augment
- [ ] TrivialAugmentation
- [ ] Repeated Augment
- [ ] TokenMix

**Comment by Ash-Lee233** [comment]
# Loss

- [ ] JSD Loss

**Comment by Ash-Lee233** [comment]
# Regularization

- [ ] Gradient Clip
- [ ] Adaptive Gradient Clip

**Comment by Ash-Lee233** [comment]
# Optimizer

- [ ] AdamP

**Comment by Ash-Lee233** [comment]
# Scheduler

- [ ] cyclic warmup cosine annealing
- [ ] MultiStep learning rate decay
